plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/types.ts

```diff
@@ -1,8 +1,41 @@
 export type Subscription = {
-	name: string;
-	status: string;
-	expiry: string;
-	autoRenew: boolean;
+	product_key: string;
+	product_id: number;
+	product_name: string;
+	product_url: string;
+	key_type: string;
+	key_type_label: string;
+	autorenew: boolean;
+	connections: string[];
+	legacy_connections: string[];
+	shares: SubscriptionShare[];
+	lifetime: boolean;
+	expires: number;
+	expired: boolean;
+	expiring: boolean;
+	sites_max: number;
+	sites_active: number;
+	maxed: boolean;
+	order_id: number;
+	product_keys_all: string[];
+	product_status: string;
+	active: boolean;
+	local: SubscriptionLocal;
+	has_updates: boolean;
```

Comment:
Thank you for figuring these out. It may be boring but it's super helpful to have this information.